fluent-plugin-cloudwatch-logs.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'fluentd', '>= 1.8.0'
+  spec.add_dependency 'fluentd', '>= 1.12.0'
```

Comment:
This patch does not use newer Fluentd feature.
So, you should stay on Fluentd 1.8.0.